src/app/search/search.component.ts

```diff
@@ -261,6 +265,12 @@ export class SearchComponent implements OnInit, OnDestroy {
     });
   }
 
+  getHistogramWidth(items: number) {
+    const rowWidth = document.getElementById('facet-row').offsetWidth;
```

Comment:
1. Pretty sure this is just using the first element it finds (see my other comment on ids). Which may be OK, but just want to make sure.
2. In general, try to avoid directly accessing the DOM like this. Instead use [ElementRef](https://angular.io/api/core/ElementRef), or according to that page, even better [Renderer2](https://angular.io/api/core/Renderer2).

See https://ucsc-cgl.atlassian.net/browse/SEAB-1766

@garyluu Will probably have better guidance on this, and you may need to get to the DOM directly for this kind of control, so this a general point where there might be an exception.

Reply:
Changed it to use ElementRef, and since it is only accessing a property and not being used to modify the DOM, this shouldn't be a security concern.